torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19610,6 +19610,7 @@ def error_inputs_mean(op_info, device, **kwargs):
         dtypes=all_types_and(torch.bool),
         promotes_int_to_float=True,
         skips=(
+            DecorateInfo(unittest.skip("Skipping - testing takes an unreasonably long time, #79528")),
```

Comment:
I'm not sure if I'm using the DecorateInfos correctly here--is this how I skip the op for all test classes? Or would it be more like
```suggestion
            DecorateInfo(unittest.skip("Skipping - testing takes an unreasonably long time, #79528"), 'TestCommon'),
```